app/config/services.yml

```diff
@@ -22,12 +22,24 @@ services:
         tags:
             - { name: twig.extension }
 
+    # Event Listeners are classes that listen to one or more specific events.
+    # Those events are defined in the tags added to the service definition.
+    # See http://symfony.com/doc/current/event_dispatcher.html#creating-an-event-listener
     app.redirect_to_preferred_locale_listener:
         class: AppBundle\EventListener\RedirectToPreferredLocaleListener
         arguments: ['@router', '%app_locales%', '%locale%']
         tags:
             - { name: kernel.event_listener, event: kernel.request, method: onKernelRequest }
 
+    # Event subscribers are similar to event listeners but they don't need service tags.
```

Comment:
This reads confusing as you still need to make use of the `kernel.event_subscriber` tag.